orcid_hub/views.py

```diff
@@ -1567,10 +1567,15 @@ def invite_organisation():
                 try:
                     org = Organisation.get(name=org_name)
                     if org.tech_contact and org.tech_contact.email != email:
-                        flash(f"The current tech.conact {org.tech_contact.name} "
+                        old_tech = User.get(id=org.tech_contact.id)
+                        old_tech.roles &= ~Role.TECHNICAL
```

Comment:
Actually, in some bizarre corner case. this user can be a tech.contact of some other organization...